recipes/cogapp/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "2.5" %}
+{% set sha256 = "b0c8efdc7d232b9e66064bae93c259946fdabad153a8c78025bc173f7245bfd7" %}
+
+package:
+  name: cogapp
+  version: {{ version }}
+
+source:
+  fn: cogapp-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/c/cogapp/cogapp-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - cogapp
+
+about:
+  home: http://nedbatchelder.com/code/cog
+  license: MIT
```

Comment:
Could we please raise an issue upstream to get a license file and have it added to `MANIFEST.in`? Please note to upstream that we are violating the terms of the MIT license by not including it.